istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -1241,7 +1241,7 @@ func bootstrapConfigCmd(ctx cli.Context) *cobra.Command {
   istioctl proxy-config bootstrap --file envoy-config.json
 
   # Show a human-readable Istio and Envoy version summary
-  istioctl proxy-config bootstrap -o short
+  istioctl proxy-config bootstrap -o short <pod-name[.namespace]>
```

Comment:
nit: maybe put name and ns before `-o short` is better, which aligns with other examples.

Reply:
```suggestion
  istioctl proxy-config bootstrap <pod-name[.namespace]> -o short
```